x-pack/plugins/apm/public/components/app/tutorials/instructions/api_key_callout.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+export function ApiKeyCallout({
+  isSuccess,
+  isError,
+  errorMessage,
+}: {
+  isSuccess: boolean;
+  isError: boolean;
+  errorMessage?: string;
+}) {
+  if (isSuccess) {
+    return (
+      <>
+        <EuiCallOut
+          title={i18n.translate(
+            'xpack.apm.tutorial.apiKey.success.calloutTitle',
+            {
+              defaultMessage: 'API key created',
+            }
+          )}
+          color="success"
+          iconType="check"
+        >
+          {i18n.translate('xpack.apm.tutorial.apiKey.success.calloutMessage', {
+            defaultMessage: `Remember to store this information in a safe place. It won't be displayed anymore after you continue`,
+          })}
+        </EuiCallOut>
+      </>
+    );
+  }
+
+  const regex = /missing the following requested privilege\(s\)/;
```

Comment:
Are we sure this will not change or be translated? Is there no http status code or error code we can match on instead?

Reply:
If there are no other identifiers I suggest displaying the error message directly to the user. This way we don't have to handle every case (it looks like you are ignoring all other error cases than "privileges missing" atm).

Reply:
This response is from the API, so i am hoping this won't be translated.
Missing privilege is a typical 403, but the API returns 500 which is the challenge here. I wanted to change it, but then since i am reusing an existing API, cannot change it as part of this change as it may cause a regression.